src/test/cloudWatchLogs/registry/logStreamRegistry.test.ts

```diff
@@ -101,17 +116,32 @@ describe('LogStreamRegistry', async function () {
         busy: false,
     }
 
-    const registeredUri = createURIFromArgs(stream1.logGroupInfo, {})
-    const shorterRegisteredUri = createURIFromArgs(simplerStream.logGroupInfo, {})
-    const missingRegisteredUri = createURIFromArgs(missingStream.logGroupInfo, {})
-    const newLineUri = createURIFromArgs(newLineStream.logGroupInfo, {})
+    const logGroupsStream: CloudWatchLogsData = {
+        data: [],
+        parameters: { streamName: 'testStreamName' },
+        logGroupInfo: {
+            groupName: 'thisIsAGroupName',
+            regionName: 'thisIsARegionCode',
+        },
+        retrieveLogsFunction: fakeSearchLogGroup,
+        busy: false,
+    }
+
+    const registeredUri = createURIFromArgs(stream1.logGroupInfo, stream1.parameters)
+    const shorterRegisteredUri = createURIFromArgs(simplerStream.logGroupInfo, simplerStream.parameters)
+    const missingRegisteredUri = createURIFromArgs(missingStream.logGroupInfo, missingStream.parameters)
+    const newLineUri = createURIFromArgs(newLineStream.logGroupInfo, newLineStream.parameters)
+    const searchLogGroupUri = createURIFromArgs(logGroupsStream.logGroupInfo, logGroupsStream.parameters)
 
     beforeEach(function () {
         map = new Map<string, CloudWatchLogsData>()
         registry = new LogStreamRegistry(new CloudWatchLogsSettings(config), map)
         registry.setLog(registeredUri, stream1)
         registry.setLog(shorterRegisteredUri, simplerStream)
         registry.setLog(newLineUri, newLineStream)
+        registry.setLog(searchLogGroupUri, logGroupsStream)
```

Comment:
Can we remove `map` in line 137 now?
Then we should remove the maps from the registry constructor parameters.